shared/constants/bridge.ts

```diff
@@ -11,3 +11,8 @@ export const ALLOWED_BRIDGE_CHAIN_IDS = [
   CHAIN_IDS.LINEA_MAINNET,
   CHAIN_IDS.BASE,
 ];
+
+export const TRUSTED_BRIDGE_SIGNER =
+  '0x533FbF047Ed13C20e263e2576e41c747206d1348';
+
+export const SIG_LEN = 130;
```

Comment:
`TRANSACTION_VERIFICATION_SIGNATURE_LENGTH` for clarity?

Reply:
If feel like that's pretty long, but if you feel strongly about it I'll change it.

Reply:
updated to something similar.